test/GetIntrinsic.js

```diff
@@ -98,15 +99,15 @@ test('dotted paths', function (t) {
 			'%ObjProto_toString%',
 			'ObjProto_toString'
 		], function (name) {
-			// eslint-disable-next-line no-extend-native
-			Object.prototype.toString = function toString() {
-				return original.apply(this, arguments);
-			};
+			defineProperty(Object.prototype, 'toString', {
```

Comment:
assigning `toString` is actually supported in SES-shim via an "overwrite mistake workaround" which is applied selectively, even though its not supported in my ses-like approximation in the test
fwiw

Reply:
I'd be fine using actual SES - that might even be easier since i could just do two complete test runs, one with and one without SES. What's the easiest way to use it?

Reply:
excellent, I'll do that then